### PR TITLE
CPDP-1557: Multiple indirect disposals company details

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/multiple_indirect_disposals_summary.scala.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.DisplayFormat
+
+@this(
+    cyaSection: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_section,
+    cyaRow: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_row,
+    cyaChange: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_change,
+    addressDisplay: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.address_display,
+)
+
+@(answers: CompleteExampleCompanyDetailsAnswers, shouldAskIfPostcodeExists: Boolean)(implicit messages:Messages)
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/multiple_indirect_disposals_summary.scala.html:33]: do you need 1shouldAskIfPostcodeExists`? That's not relevant for multiple indirect disposals

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html [hunk 2]
```diff
@@ -221,7 +222,7 @@ <h2>@messages("task-list.triage.link")</h2>
         @multipleDisposalsTriageSummary(multipleIndirect.triageAnswers, subscribedDetails.isATrust)
 
         <h2 class="heading-medium">Example company details</h2>
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html:224]: The heading here needs to be read from the messages file

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/multiple_indirect_disposals_summary.scala.html (hunk 1 above)
Comment (rambabu-posa) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/multiple_indirect_disposals_summary.scala.html:33]: yes im not using that one. just passing true and had plan to remove that flag

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsController.scala [hunk 3]
```diff
@@ -103,8 +134,15 @@ class CompanyDetailsController @Inject() (
     request: Request[_]
   ): EitherT[Future, Error, JourneyStatus] = {
     val newJourney = journey.journey.copy(
-      draftReturn = journey.draftReturn.copy(
-        companyAddress = Some(address)
+      draftReturn = journey.draftReturn.fold(
+        _.copy(exampleCompanyDetailsAnswers =
+          Some(
+            IncompleteExampleCompanyDetailsAnswers.empty.copy(
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsController.scala:140]: here you're unsetting any disposal price or acquisition price the user has entered - is that correct?

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**